bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -323,7 +327,12 @@ private synchronized int getNextReplicaIndexToReadFrom() {
         }
 
         private int getReplicaIndex(int bookieIndex) {
-            return writeSet.indexOf(bookieIndex);
+            for (int i = 0; i < writeSet.size(); i++) {
```

Comment:
make this as a method of `WriteSet`?

Reply:
writeSet already has indexOf. Removing the whole method.